tools/kagami/src/main.rs

```diff
@@ -44,8 +44,10 @@ pub enum Args {
     Crypto(Box<crypto::Args>),
     /// Generate the schema used for code generation in Iroha SDKs
     Schema(schema::Args),
-    /// Generate the default genesis block that is used in tests
+    /// Generate the genesis block that is used in tests
     Genesis(genesis::Args),
+    /// Generate the default client configuration
+    Client(client::Args),
```

Comment:
As follow up task we could generate configuration for peer as well, in this case we could use the following design: `kagami config client/peer`.